File: 01-javascript/loops/guess.js
console.log('guessing game');

var secretNumber = 42;

var guess = prompt('guess a number');

while (guess != secretNumber) {
  if (guess > secretNumber) {
    guess = prompt('failed! guess lower');
  } else {
    guess = prompt('failed! guess higher');
  }
}

console.log('not bad');

File: 04-ruby/guess/guess.rb
require 'pry'

secret_number = Random.rand(1..10)

# guess = gets.chomp.to_i

while (gets.chomp.to_i) != secret_number do
  if guess < secret_number
    puts 'you fail life guess higher'
  else
    puts 'you fail guess lower'
  end

  # guess = gets.chomp.to_i
end

puts 'you are doing ok'